fontforge/python.c

```diff
@@ -7242,6 +7242,51 @@ static int PyFF_Glyph_set_vhints(PyFF_Glyph *self,PyObject *value, void *UNUSED(
 return( PyFF_Glyph_set_hints(self,true,value));
 }
 
+static PyObject *PyFF_Glyph_get_user_decomp(PyFF_Glyph *self, void *UNUSED(closure)) {
+    if ( self->sc->user_decomp==NULL ) {
+        return( Py_BuildValue("s", "" ));
+    }
+    else {
+        char* out = u2utf8_copy(self->sc->user_decomp);
+        return( PyUnicode_DecodeUTF8(out, strlen(out), NULL) );
+        free(out);
+    }
+}
+
+static int PyFF_Glyph_set_user_decomp(PyFF_Glyph *self,PyObject *value, void *UNUSED(closure)) {
+    char *newv;
+    PyObject *temp;
+
+    if (value == Py_None) {
+        self->sc->user_decomp = NULL;
+        return 0;
+    }
+
+    if ( PyUnicode_Check(value)) {
+	/* Need to force utf8 encoding rather than accepting the "default" */
+	/*  which would happen if we treated unicode as a string */
+	temp = PyUnicode_AsUTF8String(value);
+	newv = copy( PyBytes_AsString(temp));
+	Py_DECREF(temp);
+    } else {
+        return -1;
+    }
+
+    if ( newv==NULL ) {
+        return -1;
+    }
+
+    if (strlen(newv) > 5) {
```

Comment:
5 bytes or 5 characters?

Reply:
@jtanx I thought since I'm only allowing Unicode, it would be 5 characters. Turns out that assumption was incorrect, good catch :-)

Reply:
Fixed in https://github.com/fontforge/fontforge/pull/3645/commits/a3135465fd4aa05d984d7f06870d5b13601fb71b (I hope)